plugins/linux-capture/linux-capture.c

```diff
@@ -30,6 +30,11 @@ extern void xcomposite_unload(void);
 
 bool obs_module_load(void)
 {
+	if (obs_get_platform() != OBS_PLATFORM_DEFAULT) {
+		blog(LOG_ERROR, "Cannot run with EGL platforms");
```

Comment:
linux-capture cannot run on EGL platforms

Context of where this log is coming from is important.